parachain/test-parachains/undying/collator/src/cli.rs

```diff
@@ -72,6 +119,10 @@ pub struct RunCmd {
 	/// we compute per block.
 	#[arg(long, default_value_t = 1)]
 	pub pvf_complexity: u32,
+
+	/// Configuration of the HRMP channels
+	#[clap(long)]
+	pub hrmp_params: Vec<CliHrmpChannelConfiguration>,
```

Comment:
I think sending messages should be optional.

Reply:
It is optional, e.g. no messages are sent when a vector is empty.